zoo-css-challenge/style.css

```diff
@@ -40,31 +43,40 @@ h4,
 h5 {
   /*Purple*/
   color: #6320ee;
-  margin-bottom: 1em;
+  margin-bottom: 10px;
 }
 
 h1 {
-  font-size: 250%;
+  font-size: 1rem;
 }
 
 h2 {
-  font-size: 175%;
+  font-size: 1rem;
+  color: rgb(209, 66, 22);
+  text-align: center;
```

Comment:
These CSS properties change the style for all `h2` elements on the page, but the tasks only want us to change the headings for specific sections. For example, the News heading should be centred, so we can use the selector `news h2` instead to only select the `h2` in the news section